kyo-core/shared/src/test/scala/kyoTest/typeMapTest.scala

```diff
@@ -168,4 +168,21 @@ class typeMapTest extends KyoTest:
         }
     }
 
+    ".show" - {
+        "many" in {
+            val t = TypeMap("str", true, 42, 'c').add(None).add(List[Any]()).add(Map('k' -> 'v'))
+            val expected =
+                "TypeMap(" +
+                    "Tag[java.lang.String] -> str, " +
```

Comment:
@fwbrasil maybe we should add `.showTpe` or equivalent to `Tag` so that the `Tag` is not wrapping each item. Though I don't like permuting the number of methods.

Reply:
It seems useful